src/utils/durableUtils.ts

```diff
@@ -111,19 +113,9 @@ export namespace durableUtils {
                     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access
                     let packageReferences = result?.['Project']?.['ItemGroup']?.[0]?.PackageReference ?? [];
```

Comment:
Unrelated to this PR, but there is a bug here if the package references are split into multiple item groups since we're only using the first item group. I can fix this in a separate PR.

Reply:
Ahh good catch! It should probably loop through the 'ItemGroup' and check for the 'PackageReference' instead